slack_connector.py

```diff
@@ -1294,19 +1271,27 @@ def _ask_question(self, param):
             return action_result.get_status()
 
         self.debug_print("resp json : {}".format(resp_json))
-        answer_path = resp_json.get('answer_path')
-        qid = resp_json.get('qid')
+        answer_path = resp_json.get("answer_path")
+        qid = resp_json.get("qid")
+
+        timeout_in_seconds = self._timeout * 60
+
+        if self._interval > self._timeout:
+            self.debug_print("question timeout is grater than polling interval")
+            self._interval = timeout_in_seconds
+            loop_count = 1
+        else:
+            loop_count = timeout_in_seconds / self._interval
```

Comment:
Shouldn't this be `//` so that we get an integer?

Reply:
The statement has been updated, and tested.